api/consts.py

```diff
@@ -0,0 +1,10 @@
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+DB_NAME = os.getenv("DATABASE_NAME")
+DB_USER = os.getenv("DATABASE_USER")
+DB_PASSWORD = os.getenv("DATABASE_PASSWORD")
+
+DATABASE_URL = f"mysql+pymysql://{DB_USER}:{DB_PASSWORD}@localhost:3307/{DB_NAME}"
```

Comment:
We shouldn't change the port of the db directly.

A better approach would be to define the `3307`  on a environment variable.

wdyt?